components/SubmissionComments.tsx

```diff
@@ -34,6 +39,46 @@ export const SubmissionComments: React.FC<{
     })
   }
 
+  const handleStartEditing = (index: number, initialInput: string) => {
+    if (editing !== undefined)
+      alert(
```

Comment:
This should be a modal to make the experience more pleasant. Look into react-bootstrap modal